languages/core/acf-core_nl.properties

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2016-2020 Daniel Ennis (Aikar) - MIT License
+#
+#  Permission is hereby granted, free of charge, to any person obtaining
+#  a copy of this software and associated documentation files (the
+#  "Software"), to deal in the Software without restriction, including
+#  without limitation the rights to use, copy, modify, merge, publish,
+#  distribute, sublicense, and/or sell copies of the Software, and to
+#  permit persons to whom the Software is furnished to do so, subject to
+#  the following conditions:
+#
+#  The above copyright notice and this permission notice shall be
+#  included in all copies or substantial portions of the Software.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+#  EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+#  MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+#  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+#  LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+#  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+#  WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+acf-core.permission_denied=Het spijt me, maar je hebt geen toegang om dit commando uit te voeren.
+acf-core.permission_denied_parameter=Het spijt me, maar je hebt geen toegang om dit commando uit te voeren.
+acf-core.error_generic_logged=Er is een fout opgetreden. Dit probleem is geregistreerd. Excuses voor het ongemak.
+acf-core.unknown_command=Onbekend Commando, typ <c2>/help</c2>
+acf-core.invalid_syntax=Gebruik: <c2>{command}</c2> <c3>{syntax}</c3>
+acf-core.error_prefix=Fout: {message}
+acf-core.error_performing_command=Het spijt me, maar er is een fout opgetreden bij het uitvoeren van dit commando.
+acf-core.info_message={message}
+acf-core.please_specify_one_of=Fout: Specificeer ��n van (<c2>{valid}</c2>).
+acf-core.must_be_a_number=Fout: {num} moet een getal zijn.
+acf-core.must_be_min_length=Fout: Moet minimaal {min} tekens lang zijn.
+acf-core.must_be_max_length=Fout: Mag maximaal {max} tekens lang zijn.
+acf-core.please_specify_at_most=Fout: Geef een waarde op van maximaal {max}.
+acf-core.please_specify_at_least=Fout: Geef een waarde op van minstens {min}.
+acf-core.not_allowed_on_console=Fout: De console kan deze opdracht niet uitvoeren.
```

Comment:
> Fout: De console kan deze opdracht niet uitvoeren.

What if we changed this to: "Fout: De console mag dit commando niet uitvoeren." To stay consistent with the other translations of command -> commando and at the same time we make it more specific that the console is not allowed to do it, instead of say that isnt not possible for the console to execute it?